code/modules/mob/dead/observer/observer.dm

```diff
@@ -657,6 +657,9 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		return TRUE
 
 	return FALSE
+	
+/mob/dead/observer/incapacitated()
+	return TRUE

```

Comment:
What about other machines that do not utilize NanoUI's interaction system? (although the list is shrinking, I am not yet sure it is empty yet)